source/digits_hits/src/GateGlobalActorHit.cc

```diff
@@ -0,0 +1,198 @@
+#include "GateGlobalActorHit.hh"
+#include "G4VProcess.hh"
+#include "GateRunManager.hh"
+#include "G4Track.hh"
+
+GateGlobalActorHit::GateGlobalActorHit() {}
+
+GateGlobalActorHit::~GateGlobalActorHit() {}
+
+GateGlobalActorHit::GateGlobalActorHit( const G4Step& step, const std::string& volume_name ) 
+{
+ mVolumeName = volume_name;
+ extractDataFromStep( step );
+}
+
+GateGlobalActorHit::GateGlobalActorHit( const GateGlobalActorHit& hit ) 
```

Comment:
if this is just assigning all fields, then why do you need to define it?

Reply:
Yup - it's unnecessary. Now is removed from this class.